tbtadm/controller.cpp

```diff
@@ -586,8 +586,14 @@ void tbtadm::Controller::approve(const fs::path& dir) try
     File authorized(authorizedFilename, File::Mode::Read);
     if (std::stoi(authorized.read()))
     {
-        m_out << "Already authorized\n";
+        m_out << "Already authorized, checking could we add to ACL\n";
         already_authorized = true;
+
+        if (m_sl == 2)
+        {
+            m_out << "Cannot approve already authorized devices in SL2 mode\n";
```

Comment:
I don't think this makes sense at all. Sorry.
And I think we already agreed the better solution is to have `add` command instead of changing what `approve` does.

Reply:
Yes, I can add a new command, but this would require from user some knowledge about ACL

Reply:
The same as `remove` requires (and the same as running `approve` again on already approved devices!)